examples/reminders/lib/base/data_sources/remote/reminders_remote_data_source_factory.dart

```diff
@@ -0,0 +1,23 @@
+import '../../app/config/environment_config.dart';
+import '../local/reminders_local_data_source.dart';
+import 'reminders_data_source.dart';
+import 'reminders_firebase_data_source.dart';
+
+class RemindersRemoteDataSourceFactory {
+  static RemindersDataSource withFirebaseDataSource() =>
```

Comment:
Please use the encapsulation principle, in other words hide (make private) all methods that should not be used direcly

Reply:
I have made private the two methods